pokemon-app/src/components/Pokeball.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from "react";
+import "../styles/Pokeball.css";
+
+const Pokeball = ({ isPokeball, setIsPokeball }) => {
+  useEffect(() => {
+    if (isPokeball) {
+      const timer = setTimeout(() => setIsPokeball(false), 1700);
+      return () => clearTimeout(timer);
+    }
+  }, [isPokeball, setIsPokeball]);
```

Comment:
Does `setIsPokeball` need to be a dependency here?  How often does this function change?

Reply:
That was my question too, but my VSCode gave me an yellow error description to include it. Should I delete it, sir?

Reply:
If you don't expect this prop to be reassigned at any point during execution, you can leave it out.  VSCode isn't perfect with hooks.